src/BackgroundLayerChooser/BackgroundLayerChooser.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect } from 'react';
+
+import OlOverviewMap from 'ol/control/OverviewMap';
+import OlLayerBase from 'ol/layer/Base';
+import OlLayer from 'ol/layer/Layer';
+import OlView from 'ol/View';
+import OlLayerTile from 'ol/layer/Tile';
+import OlLayerImage from 'ol/layer/Image';
+import { ObjectEvent } from 'ol/Object';
+import { getUid } from 'ol/util';
+
+import BackgroundLayerPreview from '../BackroundLayerPreview/BackgroundLayerPreview';
```

Comment:
```suggestion
import BackgroundLayerPreview from '../BackgroundLayerPreview/BackgroundLayerPreview';
```
there's a typo in the directory name (**Backround**LayerPreview)